tests/host_tools/network.py

```diff
@@ -91,48 +88,35 @@ def _init_connection(self):
         We'll keep trying to execute a remote command that can't fail
         (`/bin/true`), until we get a successful (0) exit code.
         """
-        ecode, _, _ = self._exec("true")
+        ecode, _, _ = self.execute_command("true")
         if ecode != 0:
             raise ConnectionError
 
-    def _exec(self, cmd):
-        """Private function that handles the ssh client invocation."""
+    def execute_command(self, cmd_string):
+        """Execute the command passed as a string in the ssh context."""
+        return self._exec(
+            [
+                "ssh",
+                *self.options,
+                f"{self.user}@{self.host}",
+                cmd_string,
+            ]
+        )
 
-        def _exec_raw(_cmd):
-            # pylint: disable=subprocess-run-check
-            cp = utils.run_cmd(
-                [
-                    "ssh",
-                    "-q",
-                    "-o",
-                    "ConnectTimeout=1",
-                    "-o",
-                    "StrictHostKeyChecking=no",
-                    "-o",
-                    "UserKnownHostsFile=/dev/null",
-                    "-i",
-                    self.ssh_config["ssh_key_path"],
-                    "{}@{}".format(
-                        self.ssh_config["username"], self.ssh_config["hostname"]
-                    ),
-                    _cmd,
-                ],
-                ignore_return_code=True,
-            )
+    run = execute_command
```

Comment:
How about removing this aliasing in this refactoring?

Reply:
I can just rename `s/execute_command/run/g` in the codebase, in a followup. Why I went with `run` instead is that it is much shorter to type interactively :)